backend/tracim_backend/migration/versions/fb2ae8c604ac_add_undelete_operation_type_in_enum.py

```diff
@@ -0,0 +1,59 @@
+"""add UNDELETE operation type in enum
+
+Revision ID: fb2ae8c604ac
+Revises: 9d4621f59614
+Create Date: 2020-06-23 09:56:58.775491
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "fb2ae8c604ac"
+down_revision = "9d4621f59614"
+
+old_operation_type_enum = sa.Enum("CREATED", "MODIFIED", "DELETED", name="operationtype")
+new_operation_type_enum = sa.Enum(
+    "CREATED", "MODIFIED", "DELETED", "UNDELETED", name="operationtype"
+)
+
+
+def upgrade():
+    if op.get_context().dialect.name == "postgresql":
+        # INFO - G.M - 2018-11-27 - TO modify type in postgresql, we have
```

Comment:
Isn't this weird to have old dated comment in new file ?
I don't know any good or bad practice about this.

Reply:
I'm also unsure…